include/libxnvme_spec.h

```diff
@@ -2104,6 +2306,46 @@ xnvme_spec_cmd_fpr(FILE *stream, struct xnvme_spec_cmd *cmd, int opts);
 int
 xnvme_spec_cmd_pr(struct xnvme_spec_cmd *cmd, int opts);
 
+/**
+ * Prints the given :;xnvme_spec_idfy_dir_rp to stdout
+ *
+ * @param idfy pointer to structure to print
+ * @param opts printer options, see ::xnvme_pr
+ * @return On success, the number of characters printed is returned.
+ */
+int
+xnvme_spec_drecv_idfy_pr(struct xnvme_spec_idfy_dir_rp *idfy, int opts);
+
+/**
+ * Prints the given :;xnvme_spec_streams_dir_rp to stdout
+ *
+ * @param idfy pointer to structure to print
+ * @param opts printer options, see ::xnvme_pr
+ * @return On success, the number of characters printed is returned.
+ */
+int
+xnvme_spec_drecv_srp_pr(struct xnvme_spec_streams_dir_rp *srp, int opts);
+
+/**
+ * Prints the given :;xnvme_spec_streams_dir_gs to stdout
+ *
+ * @param idfy pointer to structure to print
```

Comment:
This line should cover `sgs` instead of `idfy`

Reply:
thanks, will modify